src/code/z_actor.c

```diff
@@ -1,4 +1,37 @@
+/*
+ * File: z_actor.c
+ * Description:
+ */
```

Comment:
IMO files as big and diverse as this one need a short description and a table of contents at the top, and then section-wise documentation in addition to the usual function-wise... otherwise it'll end up with pages and pages at the top. (It'll be just like a good old GameFAQs document. :) )